docs/sources/tempo/getting-started/docker-example.md

```diff
@@ -58,16 +98,52 @@ To follow this guide, you need:
    local-tempo-1        "/tempo -config.file…"   tempo               running             0.0.0.0:3200->3200/tcp, 0.0.0.0:4317-4318->4317-4318/tcp, 0.0.0.0:9411->9411/tcp, 0.0.0.0:14268->14268/tcp
    ```
 
-1. As part of the docker compose manifest, Grafana is now accessible on port 3000. Open a browser to localhost:3000 and sign in to Grafana.
+<!-- INTERACTIVE page step1.md END -->
+
+<!-- INTERACTIVE page step2.md START -->
+
+## Explore the traces in Grafana
+
+As part of the docker compose manifest, Grafana is now accessible on port 3000. 
```

Comment:
```suggestion
As part of the Docker Compose manifest, Grafana is now accessible on port 3000. 
```

According to:  https://docs.docker.com/compose/  it should be initial caps when describing the command.

Reply:
Good catch. I found this earlier and corrected it but missed this occurrence.